docs/tutorials/testing_with_control_center.rst

```diff
@@ -208,78 +214,93 @@ the controls for setting parameters.
-  out, it will stop spinning and repeatedly beep 3 times. This is a safety feature to stop the motor if the controller is not communicating. The Control
-  Center only sends commands once per second, so 1.5s is a good choice when controlling the motor from the Control Center.
+* **Max Volts**: Set *Max Volts* to **5V**. This parameter sets the maximum voltage the module will use when in Voltage mode. Voltage mode interprets setpoints
+  as a value between 0V and this maximum voltage. If your maximum voltage is set to 5V, then a 50% setpoint will set the module to spin with 2.5V. 5V 
+  is used here because it is a fairly safe low voltage for testing, but Vertiq modules generally can handle higher voltages. Refer to the datasheet 
+  for more information on a module's electrical specifications. For more information on the *Max Volts* parameter, refer to 
+  the :ref:`throttle_mode_maximums_directions` section of the Feature Reference Manual.
+* **Timeout**: Set *Timeout* to **1.5s**. This parameter determines how long the module will wait to receive a command before timing out. When the module times 
+  out with its default settings, it will stop spinning and repeatedly beep 3 times. This is a safety feature to stop the module if the controller is not communicating. The Control
+  Center only sends commands once per second, so 1.5s is a good choice when controlling the module from the Control Center. For more information on configuring
+  the timeout settings on a module, refer to the :ref:`manual_timeout` section of the Feature Reference Manual.
 
 The figure below shows what your Tuning tab should look like after setting the parameters. Some of your default settings may be slightly different on a 
-2306, but that is fine:
+a module other than an 8108, but only the *Max Volts* and *Timeout* parametes are important for this example:
 
-.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_tuning.JPG
+.. figure:: ../_static/tutorial_images/control_center_tutorial/control_center_tuning.PNG
     :align: center
     :alt: Tuning Tab Settings
 
-    Tuning Tab Settings
+    Tuning Tab Settings on Vertiq 8108 with Firmware Version 0.0.7
 
-.. warning:: Double check that the motor is secured and there is no propeller attached before applying any of these testing parameters.
+.. warning:: Double check that the module is secured and there is no propeller attached before applying any of these testing parameters.
 
-Now at long last we can make the motor spin. Open the Testing tab, and scroll down to the bottom. At the bottom of the tab you should see 5 important testing parameters:
+Testing the Module
+*******************
+Now at long last you can make the module spin. Open the Testing tab, and scroll down to the bottom. At the bottom of the tab you should see 5 important testing parameters:
```

Comment:
I don't think you really need to say at long last

Reply:
Dramatic flair